data/json/monster_special_attacks/spells.json

```
@@ -586,6 +586,64 @@
     "min_field_intensity": 1,
     "max_field_intensity": 2
   },
+  {
+    "type": "SPELL",
+    "id": "spell_gene_sting",
+    "name": { "str": "Gene Sting" },
+    "description": "Yugg attack which causes random mutation.",
+    "effect": "attack",
+    "shape": "blast",
+    "valid_targets": [ "hostile" ],
+    "min_range": 2,
+    "max_range": 24,
```

```suggestion
    "min_range": 24,
    "max_range": 24,
```
monster spells do not use player's leveling mechanic